### PR TITLE
cleanup code in pkg/util

### PR DESCRIPTION
**Pull request by beautytiger**
**Description of your changes:**
Cleanup codebase

**Which issue is resolved by this Pull Request:**
Resolves #

**Checklist:**

- [x] Reviewed the developer guide on [Submitting a Pull Request](https://rook.io/docs/rook/master/development-flow.html#submitting-a-pull-request)
- [ ] Documentation has been updated, if necessary.
- [ ] Unit tests have been added, if necessary.
- [ ] Integration tests have been added, if necessary.
- [ ] Pending release notes updated with breaking and/or notable changes, if necessary.
- [ ] Upgrade from previous release is tested and upgrade user guide is updated, if necessary.
- [ ] Code generation (`make codegen`) has been run to update object specifications, if necessary.
- [ ] Comments have been added or updated based on the standards set in [CONTRIBUTING.md](https://github.com/rook/rook/blob/master/CONTRIBUTING.md#comments)
- [ ] Add the flag for skipping the CI if this PR does not require a build. See [here](https://github.com/rook/rook/blob/master/INSTALL.md#skip-ci) for more details.

[test ceph min]

### CODE REVIEW

**Comment by travisn** [comment]
@beautytiger Could you provide more description in the commit message for what changes were made and why they were needed?

File: pkg/util/proc/monitoredproc_test.go [hunk 1]
```diff
@@ -100,7 +99,6 @@ func TestMonitoredRestart(t *testing.T) {
 			assert.True(t, proc.monitor)
 			logger.Debugf("stop monitoring")
 			proc.monitor = false
-			break
```
Comment (BlaineEXE) [inline pkg/util/proc/monitoredproc_test.go]: This raises a question for me. This reads more like C syntax than Go. @travisn or @jbw976 is the `case iter == 0:` above supposed to `fallthrough` to the `case iter == 1:` case?

**Comment by BlaineEXE** [comment]
Also please separate out the changes to the different files into separate commits since they are each unrelated.

File: pkg/util/proc/monitoredproc_test.go (hunk 1 above)
Comment (travisn) [reply pkg/util/proc/monitoredproc_test.go]: We must not need the fallthrough for iter 0 or else the test would be failing.

**Comment by beautytiger** [comment]
> Also please separate out the changes to the different files into separate commits since they are each unrelated.

@BlaineEXE 
Sure, but to make it clear, you mean separate changes each file as a commit or as a pull request?

**Comment by BlaineEXE** [comment]
> > Also please separate out the changes to the different files into separate commits since they are each unrelated.
> 
> @BlaineEXE
> Sure, but to make it clear, you mean separate changes each file as a commit or as a pull request?

"commit"

**Comment by leseb** [comment]
@beautytiger please rebase against master.

**Comment by beautytiger** [comment]
> @beautytiger please rebase against master.

Rebase to master, please check.

**Comment by leseb** [comment]
@beautytiger thanks waiting for the CI now.

**Comment by beautytiger** [comment]
@travisn 
Thanks for your guide. I rebased the commit, please check.

**Comment by BlaineEXE** [comment]
I think the cleanup and objectstore changes should be separate PRs.

**Comment by travisn** [comment]
The objectstore commit came from #4123. @beautytiger Can you try rebasing again to remove the extra commit?

**Comment by beautytiger** [comment]
Oh, my fault. Now rebased to the right place.

**Comment by leseb** [comment]
@BlaineEXE good to go?

**Comment by BlaineEXE** [comment]
My previous request to separate the changes to the 3 files into 3 different commits (not PRs, which I clarified) hasn't been addressed still.

**Comment by leseb** [comment]
@beautytiger do you mind addressing @BlaineEXE's comments? Thanks.

**Comment by beautytiger** [comment]
@BlaineEXE @leseb 
Sorry for late reply.
Please check my lastest pr.
Thanks.

**Pull request merged**